core/controllers/question_editor.py

```diff
@@ -68,25 +71,37 @@ class QuestionSkillLinkHandler(base.BaseHandler):
     """A handler for linking and unlinking questions to or from a skill."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, question_id, skill_id):
+    def post(self, question_id, skill_ids):
         """Links a question to a skill."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
-            raise self.PageNotFoundException(
-                'The skill with the given id doesn\'t exist.')
+        try:
+            skill_ids = skill_ids.split(',')
+        except Exception:
+            raise self.PageNotFoundException
 
-        # TODO(vinitamurthi): Replace DEFAULT_SKILL_DIFFICULTY
-        # with a value passed from the frontend.
-        question_services.create_new_question_skill_link(
-            question_id, skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+        if (not isinstance(skill_ids, list) or not all([
```

Comment:
It would be a list, wouldn't it? Given that you've just constructed it using split()

Reply:
Done.